example from readme doesn't work with typescript

```
import { plot, Plot } from 'nodeplotlib';
const data: Plot[] = [{x: [1, 3, 4, 5], y: [3, 12, 1, 4], type: 'line'}];
plot(data);
```

return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/sudao.ts:12:59 - error TS2322: Type '"line"' is not assignable to type 'PlotType'.

12 const data: Plot[] = [{x: [1, 3, 4, 5], y: [3, 12, 1, 4], type: 'line'}];
                                                             ~~~~

  node_modules/@types/plotly.js/index.d.ts:1102:5
    1102     type: PlotType;
             ~~~~
    The expected type comes from property 'type' which is declared here on type 'Partial<PlotData>'

--- a/README.md
+++ b/README.md
@@ -28,7 +28,7 @@ Use with TypeScript/JavaScript:
 
 ```typescript
 import { plot, Plot } from 'nodeplotlib';
-const data: Plot[] = [{x: [1, 3, 4, 5], y: [3, 12, 1, 4], type: 'line'}];
+const data: Plot[] = [{x: [1, 3, 4, 5], y: [3, 12, 1, 4], type: 'scatter'}];
 plot(data);
 ```
 

